packages/tasks/src/tasks/chat-completion/spec/input.json

```diff
@@ -47,17 +51,42 @@
 			"type": "number",
 			"description": "The value used to modulate the logits distribution."
 		},
+		"tools": {
+			"type": "array",
+			"items": {
+				"$ref": "#/definitions/Tool"
+			},
+			"description": "The tools to use for generation."
+		},
+		"tool_choice": {
+			"type": "string",
+			"description": "The tool to use for generation."
+		},
 		"top_p": {
 			"type": "number",
 			"description": "If set to < 1, only the smallest set of most probable tokens with probabilities that add up to `top_p` or higher are kept for generation."
 		}
 	},
-	"required": ["messages"],
 	"definitions": {
```

Comment:
Unrelated to this PR but this section should be renamed `$defs` for consistency with other tasks